lib/model.js

```diff
@@ -1,32 +1,32 @@
-var Model = require('hmpo-model');
+'use strict';
 
-var util = require('util'),
-    _ = require('underscore');
+const Model = require('hmpo-model');
+const _ = require('underscore');
 
-var SessionModel = function (attrs, options) {
-    var session = options.session,
-        key = options.key;
+class SessionModel extends Model {
+    constructor(options) {
```

Comment:
Reverted the model signature back to (attrs, options). The attrs if specified are set after the model is initialized, so the values are also set onto the session object
:+1: